.gitignore

```
@@ -3,3 +3,5 @@ examples/*.js
 node_modules/
 pnpm-lock.yaml
 yarn.lock
+build/*
+coverage/*
```

`directory/*` and `directory/` do the slightly different things, the latter is more efficient, because git won't look into the directory. Because we do not need to track some files in that directories, you can remove the `*`. Also it seems that originally entries in that file was sorted, so it is better to keep that.

The all points also applicable to the `.npmignore`.

```suggestion
build/
coverage/
```

---

I didn't know this.  Thank you.  I will also update my own libraries.

---

Fixed in both